scripts/browser-specs.js

```diff
@@ -30,10 +30,27 @@ function convertSpec(spec) {
     if (spec.nightly.repository) {
         ref.repository = spec.nightly.repository;
     }
+    if (spec.standing === 'discontinued') {
```

Comment:
Where does the standing prop come from?

Reply:
The [`standing`](https://github.com/w3c/browser-specs/#standing) property is maintained in browser-specs. That property is computed automatically based on the status of the underlying spec or based on the info returned by the W3C API in many cases. For most entries contributed by browser-specs to Specref, the `discontinued` value will rather be set manually because these specs rarely come with a machine-readable discontinued flag.

Rule in browser-specs is that, instead of deleting discontinued proposals, we will now preserve them and flag them as discontinued, optionally associating them with the list of specs that replaced them.

Similarly, we will now record changes of shortnames of a spec in browser-specs through a [`formerNames`](https://github.com/w3c/browser-specs/#formernames) property. This makes it possible to create/maintain `aliasOf` properties in Specref.

Reply:
Get it. Thanks!

Reply:
Sorry for the dumb question. I was trying to get back to you quickly and reviewed from my phone so was missing the broader context.

Reply:
Well, that seemed like a very good question to me! And I hope my reply did not come through as an RTFM one, I was just trying to provide more context ;)